.github/workflows/compatibility-matrix-testing.yml

```diff
@@ -3,8 +3,8 @@ name: Compatibility Matrix Testing
 on:
   workflow_dispatch:
     inputs:
-      MM_TEST_SERVER_URL:
-        description: "The test server URL"
+      CMT_MATRIX:
+        description: "A JSON object representing the testing matrix"
```

Comment:
is this needed?

Reply:
Probably here - https://github.com/mattermost/desktop/pull/2994/files#diff-8271618e3df5dba8c8c5470a34f16e8ccbefa86b4ddf9b46a689709dfca32dc0R78